packages/app/app/reducers/equalizer.js

```diff
@@ -72,7 +63,7 @@ const getSelected = (selected, presets) => {
   if (selected.toLowerCase() === selected) {
     return selected;
   } 
-  const legacySelected = presets.find(preset => preset.label === selected);  
+  const legacySelected = presets.find(preset => preset.id === selected);  
   return legacySelected ? legacySelected.id : presets[0].id;
 };
```

Comment:
So from what I could see the label property is only used here, and is passed as the text to display in the EqualizerPresetList.

The id's follow the same naming conventions as the translation keys, so I removed the label property and used the id property as a translation key instead. You can see this in EqualizerPresetList.

But the legacySelected function scares me a bit. All the tests pass, and I couldn't find any errors while manually testing, but I'm new to this code so I might have missed something. What do you think?

Reply:
I don't know if equalizer code is properly covered by tests. You can try testing it manually, there's only a handful of presets. I think your solution is pretty good in any case.